examples/pipelineruns/pipelinerun.yaml

```diff
@@ -123,9 +123,9 @@ spec:
       description: Okay this is a hack, but I didn't feel right hard-codeing `-d1` down below
     - name: yamlPathToImage
       description: The path to the image to replace in the yaml manifest (arg to yq)
-    clusters:
-    - name: targetCluster
-      description: Not yet used, kubectl command below should use this cluster
+    #clusters:
```

Comment:
Wasn't sure where this comes from. I am guessing we should remove this altogether most likely :angel:

Reply:
oh wow! yeah feel free to remove it, this is leftover from a really old version of the spec XD

Reply:
forgot to update before removing the hold :sweat_smile:

Reply:
Could you perhaps have a test, which actually validates this behavior. Would feel better when not only the happy-paths are tested.

Reply:
@abergmeier yes, working on it (once I land somewhere 🤭)